CONTRIBUTING.md

```diff
@@ -23,7 +23,22 @@ https://gist.github.com/Chaser324/ce0505fbed06b947d962).
 
 4. If possible, add a test or two to the `tests/` directory.
 
-5. TODO: autoformatter configuration
+5. Remember to autoformat your code. 
+   
+   For Python code, we use formatters and linters listed below.
+   ```
+   pip install -r dev-requirements.txt
+   isort --check mwdb/
```

Comment:
Shouldn't it be just `isort mwdb/` and `black mwdb/` (without --check) since we want to format the code?

Reply:
Right 🤦